portfolio/src/main/java/com/google/sps/servlets/UserAndLogoutInfo.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.User;
+/** Encapsulate User info and logout url. */
+public class UserAndLogoutInfo {
+  private String email;
+  private String nickName;
```

Comment:
nit: I believe nickname is one word so the var can just be "nickname". Similar to how line 27 is user.getNickname instead of user.getNickName